### DISCUSSION TITLE
Unsuccessful so far with DCC slider on Tiguan

### DISCUSSION THREAD

**Discussion by kamold** [jilleb/MQB-FPA#78] [Q&A] [answered]
[19_3Q0907530Ax_only_FPA-Tiguan_1_20221025_163543_ODIS.zip](https://github.com/jilleb/MQB-FPA/files/9857967/19_3Q0907530Ax_only_FPA-Tiguan_1_20221025_163543_ODIS.zip)

Hi all,
I've done Passat and Superb DCC slider mods but not having much luck so far on a Tiguan.
When I apply this modified FPA dataset it seems to break drive select.  The drive select button no longer works and is no longer illuminated.
The difference maybe is that in the Tiguan it has a rotary dial as well as a centre button to invoke the drive select menu in the infotainment screen.
Has anyone had success with Tiguan?

**Comment by thaibq** [comment, marked as answer]
You edited wrong control module (DCC <>HVAC)
 Individual HMI shows the following settings: 
  Drive                         : Normal Sport Eco (2c00) 
  Steering                      : Normal Sport (c00) 
  DCC                           : Comfort Normal Sport (e00) 
  HVAC (Air Conditioning)       : A slider is displayed (feff)

**Reply by kamold** [reply]
Aha thanks for pointing out my mistake :)

**Reply by lenguyen171vn** [reply]
Do you have resolve ? can you help me do ?. My Car is Tiguan Allspace 2019

**Reply by thaibq** [reply]
Zalo đi bạn :))

**Reply by lenguyen171vn** [reply]
0922678977 Zalo của em anh nhé
